### DISCUSSION TITLE
SIMD support

### DISCUSSION THREAD

**Discussion by ib00** [odin-lang/Odin#979] [Q&A]
What is the level of SIMD support in Odin?
Are there any examples of its use?

**Comment by Tetralux** [comment]
Odin currently has experiemental SIMD support, though my understanding is there is room for improvement.
```odin
x := [?]f32{1, 1, 1, 1};
y := transmute(#simd[4]f32) x;
y += 5; // uses SIMD ops
x = transmute([4]f32) y;
```
